File: Project_UI/angular-workspace/src/app/model/Template.ts
///////////////////////////////////////////////////////////////////////////////////////////////////////
//  FILE : Templates.ts
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//
//  DESCRIPTION: Is a template ts model with constructor to encapsulate data 
//               USE this as a template to create your own ts data model 
///////////////////////////////////////////////////////////////////////////////////////////////////////
export function getString(model:Template){
    var retVal:string = "( temid:"+ model.temid;
    retVal = retVal + "  , tname:"+ model.tname;
    retVal = retVal + "  , tmess:"+ model.tmess;
    retVal = retVal + "  , tbool:"+ model.tbool + " )";
    return retVal;
}
///////////////////////////////////////////////////////////////////////////////////////////////////////
export class Template{
    public temid:number = 0;
    public tname:string = "";
    public tmess:string = "";
    public tbool:boolean = false;

    public constructor(temid:number, tname:string ,tmess:string, tbool:boolean) {
        this.temid = temid;
        this.tmess = tname;
        this.tmess = tmess;
        this.tbool = tbool;
    }
}

File: Project_API/src/main/java/com/project/api/projectapi/model/Template.java
///////////////////////////////////////////////////////////////////////////////////////////////////////    
package com.project.api.projectapi.model;

//  FILE : Template.java
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//
//  DESCRIPTION: Is a template Model with a public model to encapsulate data
//               USE this as a template to create your own model 

///////////////////////////////////////////////////////////////////////////////////////////////////////
public class Template {
    private int temid;
    private String tname;
    private String tmess;
    private Boolean tbool;

    public Template(int temid, String tname, String tmess, Boolean tbool){
        this.temid = temid;
        this.tname = tname;
        this.tmess = tmess;
        this.tbool = tbool;
    }

    public int getTemid(){
        return this.temid;
    }
    public String getTname(){
        return this.tname;
    }
    public String getTmess(){
        return this.tmess;
    }
    public Boolean getTbool(){
        return this.tbool;
    }
}
///////////////////////////////////////////////////////////////////////////////////////////////////////